src/components/widget-list-card.jsx

```diff
@@ -0,0 +1,259 @@
+    }
+
+    const toggleInCatalog = (widget) => {
+        setState(prevState => ({...prevState, widget: {...prevState.widget, in_catalog: !prevState.widget.in_catalog}}))
+    }
+
+    const toggleIsEditable = (widget) => {
+        setState(prevState => ({...prevState, widget: {...prevState.widget, is_editable: !prevState.widget.is_editable}}))
+    }
+
+    const toggleIsPlayable = (widget) => {
+        setState(prevState => ({...prevState, widget: {...prevState.widget, is_playable: !prevState.widget.is_playable}}))
+    }
+
+    const toggleIsScoreable = (widget) => {
+        setState(prevState => ({...prevState, widget: {...prevState.widget, is_scorable: !prevState.widget.is_scorable}}))
+    }
+
+    const toggleRestrictPublish = (widget) => {
+        setState(prevState => ({...prevState, widget: {...prevState.widget, restrict_publish: !prevState.widget.restrict_publish}}))
+    }
+
+    const mutateWidget = useUpdateWidget()
+
+    const saveWidget = () => {
+        const update = {
+			id: state.widget.id,
+			clean_name: state.widget.clean_name,
+			in_catalog: state.widget.in_catalog,
+			is_editable: state.widget.is_editable,
+			is_scorable: state.widget.is_scorable,
+			is_playable: state.widget.is_playable,
+			restrict_publish: state.widget.restrict_publish,
+			about: state.widget.meta_data.about,
+			excerpt: state.widget.meta_data.excerpt,
+			demo: state.widget.meta_data.demo,
+        }
+        
+        apiUpdateWidgetAdmin(update).then(response => {
+            let errorMessage = []
+            let success = false
+			for (let prop in response) {
+				const stat = response[prop]
+				if (stat !== true) {
+                    errorMessage.push(stat)
+				}
+            }
+            if (errorMessage.length == 0)
+            {
+                success = true
+            }
+            setState(prevState => ({...prevState, errorMessage: errorMessage, success: success}))
+        })
+    }
+
+    let widgetErrorsRender = null
+    if (state.errorMessage) {
+        widgetErrorsRender = state.errorMessage.map(error => <div className="error-holder">{error}</div>)
+    }
+
+    let widgetSuccessRender = null
+    if (state.success) {
+        widgetSuccessRender = <div className="success-holder">Widget Saved!</div>
+    }
+
+    let featuresRender = null
+    if (state.widget.meta_data.features) {
+        featuresRender = state.widget.meta_data.features.map((feature, i) => <li>{ feature }</li>)
+    }
+
+    let questionTypes = null
+    if (state.widget.meta_data.supported_data) {
+        questionTypes = state.widget.meta_data.supported_data.map((qtype, i) => <li>{ qtype }</li>)
+    }
+
+    let exportOptions = null
+    if (state.widget.meta_data.playdata_exporters) {
+        exportOptions = state.widget.meta_data.playdata_exporters.map((qtype, i) => <li>{ qtype }</li>)
+    }
+
+    return (
+        <li>
```

Comment:
Even though these aren't being iteratively generated as part of a map function or anything, react is still complaining about the lack of a `key` prop. I'd see about adding one to this list element.